src/config-schemas/widget-common-config.ts

```diff
@@ -0,0 +1,5 @@
+import { vitalsConfigSchema } from "./vitals-config-schema";
+
+export const widgetCommonConfigSchema = {
+  vitalsConfig: vitalsConfigSchema
+};
```

Comment:
Wait, what? This isn't common to widgets at all. This is vials-specific. The vitals widget schema shouldn't exist in patient-chart, it should be in patient-chart-widgets, with the vitals widget.